pulp_rpm/app/advisory.py

```diff
@@ -268,12 +268,23 @@ def merge_advisories(previous_advisory, added_advisory):
         # First thing to do is ensure collection-name-uniqueness
         # in the newly-merged advisory.
 
+        # If we've seen a collection-name already, create a new name
+        # by appending "_<suffix>" and rename the collection before
+        # merging.
+        # NOTE: keeping the collections separate is DELIBERATE:
```

Comment:
Does it make sense to mention as well the fact that it ensures that modular collections are handled properly (modules and their packages do not get mixed).

Reply:
def worthwhile, so Future Us/Me remembers